synced-files/.github/labels.yml

```diff
@@ -0,0 +1,137 @@
+# This is an auto-generated file. Please don't edit this file directly.
+
+# This file is automatically generated by appending two files:
+# 1. The Exercism-wide labels: defined in https://github.com/exercism/org-wide-files/blob/main/synced-files/.github/labels.yml)
+# 2. The repository-specific labels: defined in a `.appends/.github/labels.yml` file (optional)
+#
+# If you want to use additional labels besides the Exercism-wide labels,
+# don't edit this file directly but instead create a `.appends/.github/labels.yml`
+# file within this repo.
+#
+# Any changes to the repo's `.appends/.github/labels.yml`
+# file or the Exercism-wide `labels.yml` file will automatically result in this
+# file being regenerated and the labels being updated.
+
+# These are the Exercism-wide labels which are shared across all repos.
```

Comment:
I suggest a divider here to seperate the general info above from the comment below. It confused me as they flowed together.

```suggestion
# ---
#
# These are the Exercism-wide labels which are shared across all repos.
```